app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/ItemRowAdapter.java

```diff
@@ -1316,6 +1329,41 @@ public void onError(Exception exception) {
 
     }
 
+    private void retrieve(final AdditionalPartsQuery query) {
+        final ItemRowAdapter adapter = this;
+        apiClient.getValue().GetAdditionalParts(query.getItemId(), KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue().getId().toString(), new Response<ItemsResult>() {
```

Comment:
All new API related code should use the SDK instead of the apiclient

Reply:
I'm running into some difficulties with this, I'm hoping due to my inexperience with Jellyfin development.

I think I have retrieving the additional parts with the SDK working, but the BaseRowItem class expects the old BaseItemDto class, and I only see a way to convert from the apiclient BaseItemDto class to SDK BaseItemDto. Is there a way to go the other way, or if there's not, should I implement one? Or is there something entirely different that I should be doing?

Reply:
Yeah looks like using the SDK here requires a lot of changes, so we can using the apiclient here for now is fine.